.github/workflows/testing.yml

```diff
@@ -63,3 +60,54 @@ jobs:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           make coveralls
+  run-tests-ee:
+    if: github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+
+    runs-on: ubuntu-latest
+
+    env:
+      TEST_TNT_EE: 1
+
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk-version: ["2.10.0-beta2-59-gc51e2ba67-r469"]
```

Comment:
1.10, 2.8 added

> and it looks like it would be better to separate that commit from SSL support introduction

Ok, i'll fetch changes inside .github/ into a separate commi before merge.